ClangSharpPInvokeGenerator/Program.cs

```diff
@@ -100,20 +105,35 @@ public static int Run(InvocationContext context)
                 foreach (var file in files)
                 {
                     var translationUnitError = CXTranslationUnit.Parse(createIndex, file, arr, Array.Empty<CXUnsavedFile>(), translationFlags, out CXTranslationUnit translationUnit);
+                    bool skipProcessing = false;
 
                     if (translationUnitError != CXErrorCode.CXError_Success)
                     {
-                        Console.WriteLine($"Error: '{translationUnitError}' for '{file}'.");
-                        var numDiagnostics = translationUnit.NumDiagnostics;
+                        Console.WriteLine($"Error: Parsing failed for '{file}' due to '{translationUnitError}'.");
```

Comment:
If the parsing failed, there is no guarantee the translation unit is even valid to get diagnostics